clients/mockclients/repo_client.go

```diff
@@ -20,109 +20,108 @@
 package mockrepo
 
 import (
-	reflect "reflect"
-
 	gomock "github.com/golang/mock/gomock"
 	clients "github.com/ossf/scorecard/v4/clients"
+	reflect "reflect"
```

Comment:
the linter is going to complain if this line is at the end, I think

Reply:
this was generated automaticaly

Reply:
but it is fixed :)